tests/testthat/testImportCNA.GISTIC2.R

```diff
@@ -0,0 +1,45 @@
+#
+# test importCNA.GISTIC2.R
+#
+library(testthat)
+context( "import rCNA data from GISTIC2 files")
+#
+#
+
+
+test_that("File does not exist", {
+    expect_error(
+        importCNA.GISTIC2(NULL,"inst/extdata/dCNA")
```

Comment:
better to add the **actual** error message to the test - since in your case ```"inst/extdata/dCNA``` should **also** have produced an error and that would have been obscured.